src/components/Interview/Interview.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import './Interview.css';
+
```

Comment:
That's not where the adapter goes... The tests are failing because you have to have tests in the files for them to pass, so I deleted them until there's something to go in them.

Reply:
Ahhh, gotcha. Good thinking, Aurie.

Reply:
Ah, sorry I misread the error in Travis.